src/main/scala/mesosphere/marathon/core/group/impl/GroupManagerImpl.scala

```diff
@@ -151,9 +152,9 @@ class GroupManagerImpl(
         eventStream.publish(GroupChangeSuccess(id, version.toString))
       case Success(Left(_)) =>
         ()
-      case Failure(ex: AccessDeniedException) =>
+      case Failure(RejectionException(_: Rejection.AccessDeniedRejection)) =>
```

Comment:
hmm I thought that rejections are just something API related, it's kind of unexpected that it bubbles down to the core too...

Reply:
Yeah, converting this to a rejection I think surfaced it :)

It should just return a `Left`, but, at some point I had to call it quits or this patch would get really large.

Reply:
I'd solve it right now, but this patch is getting too large.

I'd put a TODO, but people don't like TODOs.

I'd create a JIRA, but realistically it will not get prioritized.

Reply:
I went ahead and added a note about this in MARATHON-8202